src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(SourceBuildTargetFrameworks);net472</TargetFrameworks>
+    <!-- We'll always be running our build host with the same host that is used to launch the language server process directly, so we don't need to create another one -->
+    <UseAppHost>false</UseAppHost>
+    <!-- List of runtime identifiers that we want to publish an executable for -->
+    <RuntimeIdentifiers>win-x64;win-x86;win-arm64;linux-x64;linux-arm64;alpine-x64;alpine-arm64;osx-x64;osx-arm64</RuntimeIdentifiers>
```

Comment:
are we building an r2r build host?  If so we need additional targets to actually publish as r2r.  If not we don't need to list these I don't think.

Reply:
We're not, but without this I was running into issues since the ready-to-run versions were passing along the RuntimeIndentifier somewhere. It might have been because technically this is an executable, not a library. I can poke more at the MSBuild if you have concerns here.

Reply:
To me these are misleading - since we aren't producing an R2R executable (or even publishing with runtime identifiers) its weird to list them in the project.  Would prefer to remove if we can.

Reply:
Let me remove them again and see what broke.